### PR TITLE
make IOR SKU independant

### PR DESCRIPTION
**Pull request by xpillons**
initially IOR was built to be only run from HB/HC. With these changes it can now be used from any kind of SKUs

### CODE REVIEW

File: apps/ior/readme.md [hunk 1]
```diff
@@ -1,27 +1,58 @@
...
-This requires ior to be built on HB or HC sku's with CentOS-HPC 7.6 (using mpi/mpich-3.3). See one of the examples for building a Cluster with HB or HC skus and PBS. (e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md))
+IOR can be built on any kind of VM SKU as long as it uses the CentOS-HPC 7.6 image. See one of the examples for building a Cluster with HB or HC skus and PBS. (e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md))
+
+## Pre-requisites
+
+The pre-requisites for running IOR are those :
+
+- jq
+- mpich-3.3 for HB/HC
+- mpich-2.2-devel for others skus
```
Comment (garvct) [inline apps/ior/readme.md]: Should this be mpich-3.2-devel?

Comment (xpillons) [reply apps/ior/readme.md]: Yes this is a typo

**Comment by xpillons** [comment]
> If you build on  non (HB,HC) sku with mpich-3.2 and run on HB or HB with mpi/mpich-3.3, does this work ok?

i guess, if not mpich-3.2 should be installed as well

**Comment by xpillons** [comment]
@garvct I've successfully validated that building on DS_v3 can then be run from HB just by loading the right MPI package. So I'm going to merge that branch.

**Pull request merged**